roles/high_availability/tasks/main.yml

```diff
@@ -105,7 +105,7 @@
 
     - name: command █ Set properties
       command: "pcs property set {{ item.name }}={{ item.value }}"
-      loop: "{{phigh_availability_pcs_propert | default([]) }}"
+      loop: "{{high_availability_pcs_propert | default([]) }}"
```

Comment:
Still tiny typo:
```
      loop: "{{high_availability_pcs_property | default([]) }}"
```

Reply:
I forgot to save the file 😄 

Thanks a lot for checking that one. Let me commit.

Reply:
Should be ok now